sencha-workspace/SlateDemonstrationsTeacher/sass/src/view/main/StudentsProgressGrid.scss

```diff
@@ -17,10 +16,10 @@
 
 @for $n from 8 through 13 {
     .cbl-level-#{$n} {
-        $level-color:        nth($cbl-level-colors,        ($n - 7));
-        $level-color-light:  nth($cbl-level-colors-light,  ($n - 7));
-        $level-color-medium: nth($cbl-level-colors-medium, ($n - 7));
-        $level-color-dark:   nth($cbl-level-colors-dark,   ($n - 7));
+        $level-color:        nth($level-colors,        ($n - 7));
+        $level-color-light:  nth($level-colors-light,  ($n - 7));
+        $level-color-medium: nth($level-colors-medium, ($n - 7));
+        $level-color-dark:   nth($level-colors-dark,   ($n - 7));
```

Comment:
@aliyeysides did you make all the SCSS changes in this file yourself or port them from something @ryon did?